src/create-claim-group.js

```diff
@@ -1,26 +1,26 @@
 #!/usr/bin/env node
 'use strict'
 const config = require('config')
+require('dotenv').config()
```

Comment:
using `config` and `dotenv` is not good practice. Which I learnt after making few mistakes my self.